### DISCUSSION TITLE
Loading data to form components: selects or autocompletes in parts

### DISCUSSION THREAD

**Discussion by AniaKru95** [vuetifyjs/vuetify#16511] [Q&A]
Hello,

I'm wondering, is it possible to fill autocomplete component by data chunks?
I have huge list from backend and it takes a while before all data is displayed in autocomplete.

Is it possible to get, for example 10 elements, and after scroll, load next 10 elements?
Something like infinity scroll in form components.

I would be appreciate it for your help. 

(I use vue3 and vuetify 3.1.1)

**Comment by AniaKru95** [comment]
I found virtual-scroll-component, now only in documentation lab's section.
https://next.vuetifyjs.com/en/labs/virtual-scroller/

Is it possible to use it with select or autocomplete?

**Comment by velly** [comment]
I think this is a common question for rendering large items in select or autocomplete.

**Reply by velly** [reply]
Provide a temp solution that is  watching the menu.
When menu inserted to overlay container, you can add a scroll event listener to load displayed items lazily.
